presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeGeneralQueries.java

```diff
@@ -41,6 +42,21 @@ protected void assertQuery(String sql)
         PrestoSparkNativeQueryRunnerUtils.assertShuffleMetadata();
     }
 
+    @Test
+    public void testNullIf()
+    {
+        assertQuery("SELECT NULLIF(totalprice, 0) FROM (SELECT SUM(extendedprice) AS totalprice FROM lineitem WHERE shipdate >= '1995-09-01')");
```

Comment:
These tests look very similar to me. What's the motivation for adding so many nearly identical tests?

Reply:
Permutation combination of bigint, double and empty rows (see date filter).